### DISCUSSION TITLE
（疑问）移动端评论显示问题

### DISCUSSION THREAD

**Discussion by aaro-n** [ArtalkJS/Artalk#307] [Q&A] [answered]
# 运行及引用环境
后端版本：v2.3.0
前端js及CSS（ArtalkLite）：2.4.3
后端部署环境：Docker
# 遇到问题：
在Windows chrome访问博客，评论显示正常
![PC访问](https://user-images.githubusercontent.com/45537680/200099317-316980fc-392d-4b82-8c28-75d762e471e8.png)
使用Chrome 模拟移动端访问（F12，尺寸iPad mini）,感觉显示有问题：
![模拟移动端访问](https://user-images.githubusercontent.com/45537680/200099414-2fa4c78b-a81e-4d74-88b1-a6bbd1f691e1.png)
移动端优先显示回复，没有PC端显示顺眼
# 其他
最近添加的控制中心，显示也不顺眼。（直接用PC访问，没有模拟移动端。）
![控制中心](https://user-images.githubusercontent.com/45537680/200099596-f898b315-e6f4-4354-aa0f-ddf7fc0c740c.png)

**Comment by qwqcode** [comment, marked as answer]
https://artalk.js.org/guide/frontend/config.html#flatmode

**Comment by aaro-n** [comment]
明白了，谢谢。
